CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+
+3.Express Interest: If you want to work on the issue after it's been reviewed, mention this in your issue description.
+
+### Working on an Issue
+
+Before starting work on an existing issue, please follow these steps:
+
+1. **Request Assignment**: Comment on the issue asking for it to be assigned to you.
+2. **Prepare for Assignment**:
+    - Confirm that you have read the `CONTRIBUTING.md`.
+    - Ensure you have a functional development environment (you can build and run the project).
+    - Describe your intended approach to solving the issue.
+    - Include these points in one or more comments.
+3. **Start Work**: Begin working on the issue only after it has been assigned to you.
+4. **Avoid Conflicts**: Only start working on the issue and open a Pull Request after it has been assigned to you. This prevents confusion, duplicate work, and potential conflicts among contributors.
+5. **Reference the Issue**: In your Pull Request, reference the issue (e.g., `This PR fixes #1234`). This ensures the issue is automatically closed when your Pull Request is merged.
+
+
+> **Notes:**
+>
+> - **Check the Assignees Box**: Before requesting to be assigned to an issue, check the `Assignees` box at the top of the page to see if it has already been assigned to someone else. If it has a current assignee but appears to be inactive, politely ask them if they are still working on it or if you might collaborate with them.
+> - **Requesting Assignment**: Only request to be assigned to an issue if you know how to work on it.
+> - **Seek Clarity**: If an issue is unclear, ask questions to get more information before requesting to be assigned. Avoid asking vague questions like "what do I do next?" or "how do I fix this?"
+> - **Collaboration**: An issue can be assigned to multiple people if all parties agree to collaborate on it. The pull request can contain commits from different collaborators.
+> - **Inactivity**: Any issue with no activity for 2 weeks will be unassigned and re-assigned to someone else.
+
+
+## Reviewing Pull Requests
+
+We encourage everyone to review pull requests. It's a fantastic way to learn, network, and support each other.
+
+### DOs
+
+- Use inline comments to provide clear explanations for your suggestions.
+- Offer inline suggestions to propose changes.
+- Exercise patience and empathy when providing feedback on others' work.
+
+### DON'Ts
+
+- Avoid repeating feedback, as it creates unnecessary noise. Check the existing conversation and use GitHub reactions to agree or disagree with a comment.
+- Do not approve pull requests without proper review just to improve your GitHub contributors graph.
+
+
+## Style Guide
+
+The theupdateframework.io site is built using HTML and Javascript.
+
+Check out the set of contributing guides available at https://theupdateframework.readthedocs.io/en/latest/CONTRIBUTING.html
```

Comment:
Also, why referencing `python-tuf`'s [Instructions for contributors](https://theupdateframework.readthedocs.io/en/latest/CONTRIBUTING.html) here? That page contains 

- DCO
- Testing 
- Unit tests
- Coverage
- Auto formatting

Which completely based on `python-tuf` repo, does not relates to `theupdateframework.io` repository. 

Thank you 👍

Reply:
Hi @h4l0gen,

Thank you for the feedback! 👍 You’re right, referencing the python-tuf’s instructions isn't relevant here. I’ll update the contributing guide to better reflect the specifics of the theupdateframework.io repository. 

Thanks for pointing that out! 🙌

Reply:
Please just wait for official maintainer's review, before making any commit. I am not official maintainer, so there review is must required on if my review is correct or not. I hope you understand. Thank you 👍

Reply:
Hi @h4l0gen ,

Of course, I understand. I'll wait for the official maintainers' review before making any changes.